api/autonomous_car/fixtures/successful_car_route_with_obsticals.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
typo again: *obstacles

at least you're consistent

Reply:
LOL whoopsy